pkg/strings/strings.go

```diff
@@ -6,8 +6,14 @@ import (
 )
 
 func Contains(input []string, match string) bool {
+	return IsPresentInListSatisfying(input, match, func(listElement string, toMatch string) bool {
+		return listElement == toMatch
+	})
+}
+
+func IsPresentInListSatisfying(input []string, toMatch string, matchingPredicate func(string, string) bool) bool {
```

Comment:
While looking for naming ideas in go standard packages (usual name for such a function is `ContainsFunc`), I realized the [`slices`](https://pkg.go.dev/slices#ContainsFunc) package has both `Contains` and `ContainsFunc`, we can use these instead.

Reply:
Oh, thanks a lot! I'm sorry I should've checked this.